galleries/examples/text_labels_and_annotations/demo_annotation_box.py

```diff
@@ -15,8 +15,16 @@
 from matplotlib.cbook import get_sample_data
 from matplotlib.offsetbox import (AnnotationBbox, DrawingArea, OffsetImage,
                                   TextArea)
-from matplotlib.patches import Circle
+from matplotlib.patches import Annulus, Circle
 
+# %%%%
+# Text
+# ====
+#
+# `.AnnotationBbox` supports positioning annotations relative to data, Artists, and
```

Comment:
Can you please add a sentence that `annotate` should be used for simple cases, and possibly explain what this can do more than a simple `annotate` call?